feature-utils/poly-look/src/css/colors.css

```diff
@@ -30,6 +30,8 @@
   --color-gray-700: #525e72;
   --color-gray-800: #3e495b;
 
+  --color-blue: #3BA6FF;
```

Comment:
Does it have a number value?

Reply:
I couldn't find it and since this is mostly a speed run, I've figure it's good enough for now. If you do know the numbering or where to find it pls le me know.